live/root/root/.local/bin/gnome-kiosk-script

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Start a browser to connect to Agama's web user interface skipping the authentication.
+# This script is invoked as a part of gnome-kiosk startup procedure. See /usr/bin/gnome-kiosk-script
+# which is at the end of the chain and invokes this script.
+
+TOKEN_FILE=/run/agama/token
+TOKEN=$(cat $TOKEN_FILE)
+PREFS=$HOME/.mozilla/firefox/profile/user.js
+
+sed -e "s/__HOMEPAGE__/http:\/\/localhost\/login?token=$TOKEN/" $PREFS.template > $PREFS
+firefox --kiosk --profile $HOME/.mozilla/firefox/profile
+# Here, we would like to see
```

Comment:
I guess this comment is outdated, right?

Reply:
hopefully ;-)